### ISSUE TITLE
Houdini examples are not working with the HDA unity plugin on missing nodes

### ISSUE THREAD

**Issue by artigee** [Unity-Technologies/VFXToolbox#1]
Load warnings for E:/VFXToolbox/DCC-Tools/Houdini/Examples/VectorFields/vectorFields.hipnc

Warning:     "/obj/VectorFields/volumetile9" using incomplete asset definition (full definition not found).
             "/obj/VectorFields/volumetile8" using incomplete asset definition (full definition not found).
             "/obj/VectorFields/volumetile7" using incomplete asset definition (full definition not found).
             "/obj/VectorFields/volumetile6" using incomplete asset definition (full definition not found).
             "/obj/VectorFields/volumetile5" using incomplete asset definition (full definition not found).
             "/obj/VectorFields/volumetile4" using incomplete asset definition (full definition not found).
             "/obj/VectorFields/volumetile3" using incomplete asset definition (full definition not found).
             "/obj/VectorFields/volumetile2" using incomplete asset definition (full definition not found).
             "/obj/VectorFields/volume_previs1" using incomplete asset definition (full definition not found).
             "/obj/VectorFields/volumetile1" using incomplete asset definition (full definition not found).
             The following node types are using incomplete asset definitions:
    Sop/volumetile
    Sop/volume_previs

any updates on these errors?
Thanks in advance

**Comment by artigee** [comment]
and also the tool seems to be restriced to only support on cube with x,y,z same resolution which seems to be understandable due to performance issue and need for 3D texture format for sure. Currently, pure vector field data ( like from pyro sim cache into volume for instance ) only supported, which seems to be quite restricted. is there any improvement we can expect? thanks in advance

**Comment by peeweek** [comment]
Indeed, there were some custom operators not embedded in the sample file, this should be fixed in (412b220771f2b5e64b1f4a42bb3ced631bc65d18)

Please note that you still need to install the Unity_VFX_Tools.hda into *Scanned Asset Library Definitions* (you can do it by using File/Import/Houdini Digital Asset, selecting the .hda and setting the "Install Library to:" dropdown)

**Comment by artigee** [comment]
Thanks for the update and do appreciate this utility. Super helpful! Very impressed by the latest updates on effect graph and keep up with great works!
